spec/tests/helpers/perf_run_helper_spec.rb

```diff
@@ -793,5 +793,67 @@ class PerfRunHelperClass
       end
     end
   end
+  describe "#save_average_transaction_time" do
+    let(:host) do
+      Beaker::Host.create("localhost", { role: "database" }, logger: @logger)
+    end
+    before { subject.instance_variable_set(:@start_epoch, 0) }
+    before { subject.instance_variable_set(:@end_epoch, 946_684_799) }
+    let(:file) { Tempfile.new(described_class.to_s) }
+    let(:result) { Beaker::Result.new(host, "") }
```

Comment:
should this be in a before or something to ensure it is new for each test?

Reply:
This is just to get a Beaker::Result object.  Each test explicitly defines the `stdout` for it, so I don't think it is an issue.